src/main/routes/routes.ts

```diff
@@ -139,6 +139,11 @@ export default function (app: Application): void {
     app.get('/iac-daily-list', app.locals.container.cradle.iacDailyListController.get);
     app.get('/primary-health-list', app.locals.container.cradle.tribunalNationalListsController.get);
     app.get('/magistrates-public-list', app.locals.container.cradle.magistratesPublicListController.get);
+    app.get('/media-account-rejection-reasons', app.locals.container.cradle.mediaAccountRejectionReasonsController.get);
```

Comment:
Can we have these 2 routes n the same place as the existing media account routes like '/media-account-rejection' and '/media-account-rejection-confirmation'?
Also the other media account routes have the permission enforced using " isPermittedMediaAccount" so this should be the same

Reply:
Done now.